### ISSUE TITLE
The viewer (viewer.scuttlebot.io) does not show profiles

### ISSUE THREAD

**Issue by steelman** [ssbc/scuttlebot#470]
(I am sorry if this isn't the right place to report bugs about the viewer)

# Steps to reproduce:

1. Post a message.
2. Get its external [link](https://viewer.scuttlebot.io/%25GWnRLzfXjIUnkeAMVao8IrELNdtDa9RU5jCh5Cg5qfc%3D.sha256) 
3. Open in a browser.
4. Click the [@steelman](https://viewer.scuttlebot.io/@NBeFgSSWNtEo0yALqRoKsIF0U0rCFmCXwEbNUmIygeQ=.ed25519) link right next to the avatar.

# Expected result:

Some kind of a user profile/timeline is presented.

# Actual result

"Forbidden"

**Comment by cryptix** [comment]
yes, the public gateways need some love. I think they get hammered with too many requests.

We should collect good caching configurations for nginx/apache/you name it to relife the nodejs processes from the load.

**Comment by steelman** [comment]
Now even the [post](https://viewer.scuttlebot.io/%25GWnRLzfXjIUnkeAMVao8IrELNdtDa9RU5jCh5Cg5qfc%3D.sha256) is not available. Instead the following message is displayed:
> User has not chosen to be hosted publicly

**Comment by dominictarr** [comment]
you need to opt-in to public web hosting. hmm, trying to find the bit in the code where that happens:
https://github.com/ssbc/ssb-viewer/blob/master/index.js (can't see it!)

aha, this post explains it `%Fli0kemExYgucD1fLJm50qD5WptN6wZpsfIXddQpvNI=.sha256`

---

## how to change your publicWebHosting

to change you public web hosting, you have to write an `about` message with a `publicWebHosting` field.

using sbot (to install sbot, first install node with nvm, then run npm install -g scuttlebot)

```
sbot publish --type about --about "@your.public.id.here" --publicWebHosting
```

**Comment by steelman** [comment]
Doable, but why patchwork can't do it? Thank, you.

**Comment by dominictarr** [comment]
well, someone just needs to decide to add that ;) then do it

**Comment by steelman** [comment]
Any one there? If this hasn't been implemented please keep it open with the low-hanging-fruit label for those who may want to start contributing.
